doc/rst/platforms/infiniband.rst

```diff
@@ -105,24 +105,27 @@ Selecting a Spawner
 Under the covers, ``gasnetrun_ibv``-based launchers must figure out
 how to spawn jobs and get them up and running on the compute nodes.
 GASNet's three ways of doing this on InfiniBand systems are ``ssh``,
-``pmi``, and ``mpi``.  When MPI is detected at configure time, it will
-be selected as the default, but we recommend using one of the other
-options if possible.
-
-* ``ssh``: Based on our experience, we recommend launching with the
-  ``ssh`` option, if possible.  This requires the ability to ``ssh``
-  to the system's compute nodes, which is not supported by all
-  systems, depending on how they are configured.  See the following
-  sub-section for details on this option.
+``pmi``, and ``mpi``, described just below.  When MPI is detected at
+configure time, it will be selected as the default, but we recommend
+using one of the other options if possible.  This can be done by
+setting the ``GASNET_IBV_SPAWNER`` environment variable, whose legal
+values are:
+
+* ``ssh``: Based on our experience, this is the preferred option, when
+  possible.  This requires the ability to ``ssh`` to the system's
+  compute nodes, which is not supported by all systems, depending on
+  how they are configured.  See the following sub-section for details
+  on this option.
   
 * ``pmi``: When GASNet's configure step detects a PMI-capable job
   scheduler like Slurm, ``pmi`` can be the next best choice because it
   often "just works" and can reduce overhead compared to ``mpi``.
 
 * ``mpi``: When the previous cases are not options, ``mpi`` serves as
-  a reasonable last resort.  Note that it can incur a performance
-  penalty because MPI will be running concurrently with GASNet.  See
-  the second subsection below for tips on this option.
+  a reasonable last resort.  Note that it may, depending on its
+  configuration, incur a performance penalty due to competition
+  between MPI and GASNet for limited communication resources.  See the
+  second subsection below for best practices when using this option.
```

Comment:
Might be worth a link _here_ to https://bitbucket.org/berkeleylab/gasnet/src/master/other/mpi-spawner/README as part of a mention that the environment variable `MPIRUN_CMD` is used to configure how jobs are started; or just as "for more information".  This would be in addition to the link below where the topic is resource issues.  
Same for the omnipath document.